civitai-downloader-v2/src/core/download/manager.py

```
@@ -589,6 +598,64 @@ def prioritize_safetensors(self, files: List[FileInfo]) -> List[FileInfo]:
+                size=0  # Will be determined during download
+            )
+            
+            # Set output path
+            if output_dir:
+                output_path = Path(output_dir) / file_info.name
+            else:
+                output_path = self.default_output_dir / file_info.name
+            
+            # Create and start download task
+            task_id = self.create_download_task(file_info, output_path)
+            success = await self.start_download(task_id)
```

Ruff (0.12.2)

Local variable `success` is assigned to but never used

Remove assignment to unused variable `success`

(F841)